apps/interactive-calibration/main.cpp

```diff
@@ -32,6 +32,8 @@ const std::string keys  =
         "{dst      | 295     | Distance between white and black parts of daulCircles template}"
         "{w        |         | Width of template (in corners or circles)}"
         "{h        |         | Height of template (in corners or circles)}"
+        "{chd      | 0       | number of predefined charuco dictionary}"
```

Comment:
Name of pre-defined dictionary? I expect to see list of available Arcuo dict names and string values as input. Please stress that it's for ChAruco board only.